exporter/debugexporter/config.go

```diff
@@ -0,0 +1,44 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package debugexporter // import "go.opentelemetry.io/collector/exporter/debugexporter"
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configtelemetry"
+)
+
+var (
+	// supportedLevels in this exporter's configuration.
+	// configtelemetry.LevelNone and other future values are not supported.
+	supportedLevels map[configtelemetry.Level]struct{} = map[configtelemetry.Level]struct{}{
+		configtelemetry.LevelBasic:    {},
+		configtelemetry.LevelNormal:   {},
+		configtelemetry.LevelDetailed: {},
+	}
+)
+
+// Config defines configuration for logging exporter.
+type Config struct {
```

Comment:
You didn't move the config to the internal to avoid adding the deprecated `logLevel` field?

Reply:
Right, i figured there's no need to bring over loglevel in the debug exporter